src/pages/SignUp/AddMoreUserData/AddMoreUserData.js

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import AddMoreUserInfoData from './AddMoreUserInfoData';
+import styled from 'styled-components';
+
+function AddMoreUserData() {
+  const [inputMoreInfo, setInputMoreInfo] = useState({
+    email: { value: localStorage.getItem('user_email'), valid: '' },
+    nickname: { value: localStorage.getItem('user_nickname'), valid: '' },
+    proSelf: { value: '', valid: '' },
+  });
+
+  let history = useHistory();
+
+  const validationFunc = {
+    email: value => value.includes('@'),
+    nickname: value => value.length >= 1,
+    proSelf: value => value === '전문가' || value === '셀프',
+  };
+
+  const handleMoreInput = e => {
+    const valid = validationFunc[e.target.name](e.target.value);
+    setInputMoreInfo({
+      ...inputMoreInfo,
+      [e.target.name]: { value: e.target.value, valid },
+    });
+  };
+
+  useEffect(() => {}, [inputMoreInfo]);
+
+  const buttonActive = e => {
+    const checkvalidationArray = Object.values(inputMoreInfo).every(
+      v => v.valid
+    );
+    if (checkvalidationArray === true) {
+      sendMoreUserInfo();
+    }
+  };
+
+  const sendMoreUserInfo = () => {
+    const { email, nickname, proSelf } = inputMoreInfo;
+    const is_expert = proSelf.value === '전문가';
+    const tokenValue = localStorage.getItem('kakao_account_token');
+    fetch(`http://10.58.7.181:8000/users/user`, {
+      method: 'POST',
+      body: JSON.stringify({
+        access_token: tokenValue,
+        email: email.value,
+        nickname: nickname.value,
+        is_expert: is_expert,
```

Comment:
```suggestion
        access_token: localStorage.getItem('kakao_account_token');
        email: email.value,
        nickname: nickname.value,
        is_expert: proSelf.value === '전문가';
```
여러군데서 쓰이는 값이 아닌 것 같아서 따로 변수 선언하지말고 바로 적어주셔도 될 것 같습니다!

Reply:
네 말씀 해주신 부분 수정하였습니다. 
```js
        access_token: localStorage.getItem('kakao_account_token'),
        email: email.value,
        nickname: nickname.value,
        is_expert: proSelf.value === '전문가',
```